src/EnergyPlus/WaterManager.cc

```diff
@@ -1627,6 +1690,49 @@ namespace WaterManager {
         }
     }
 
+    void ReportRainfall(EnergyPlusData &state)
+    {
+        Array1D_string const Months(12, {"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"});
```

Comment:
Use a constexpr std::array if no know both the size and contents of the list at compile time.

Reply:
Thanks. I changed the Months to constexpr std::array.